resources/opdb/converter.go

```diff
@@ -27,6 +28,17 @@ func fromModelToDatabaseRequest(model databaseResourceModel, ctx context.Context
 	req.StorageType = opdbmodels.StorageType(model.StorageType.ValueString())
 	req.DisableExternalDB = !model.DisableExternalDB.IsNull() && model.DisableExternalDB.ValueBool()
 
+	req.DisableMultiAz = !model.DisableMultiAz.IsNull() && model.DisableMultiAz.ValueBool()
```

Comment:
ValueBool() returns false if its null or unknown, so if I understand correctly it will be false, eventhough you want this to be true when it is null

Reply:
I think we want that to be false in that case, So multi AZ deployment won't be disabled by default.
Right @petersomogyi ?

Reply:
Oh, I think I misunderstood it.

Reply:
If it is supposed to be false, then it is working as intended however I think the first part is unnecessary:
`!true && false`

Reply:
Yes thats right, I removed the first part from this and from the DisableExternalDB too.